examples/tutorials/color-figure.py

```diff
@@ -0,0 +1,95 @@
+"""
+Adding a color palette to a figure
```

Comment:
Not sure if we should use "color palette" or "color bar" here.

Reply:
My intended purpose for this tutorial was to show how to add color to a figure using `gridimage`, which is why I put "color palette" at the top. I included adding a color bar in the guide because that seemed like a logical addition to show how to add a legend for the color scheme. My opinion is to leave it as "color palette" but I'm fine with whatever you think is the better option.

Reply:
The description of the [`grdimage`](https://docs.generic-mapping-tools.org/dev/grdimage.html) module is 
> Project and plot grids or images

Is it better to use `Plot grids` as the title?

Reply:
I changed the title

Reply:
How about "Coloring grids"?